Tests/test_mixtures_and_components_combinatorially.py

```diff
@@ -108,8 +117,8 @@ def test_components_in_mixtures(self):
 					args["parameters"] = dict(self.parameters)
 					M = mixture(**args, components = [C])
 
-					if C not in M.components:
-						raise AttributeError(f"{C} not found in {M}.components")
+					if M.get_component(component = C) is None:
+						raise AttributeError(f"{C} not found in M.components={M.components}")
 
 					CRN = M.compile_crn()
```

Comment:
Once you merge #131, you'll get SBML validation here. Also, if there are other places that you are creating a CRN model it would be good if you could add that same SBML validation code.

Reply:
I think this is the only place where CRNs are compiled - will make sure we get SBML validation.